frontend/src/styles/globals.scss

```diff
@@ -22,6 +22,7 @@ a {
 /*
   Josh's Custom CSS Reset
   https://www.joshwcomeau.com/css/custom-css-reset/
+  TODO: Try changes from June 2023 and later
```

Comment:
I opened [MPP-3959](https://mozilla-hub.atlassian.net/browse/MPP-3959) for this `TODO`